dev-docs/releasing.adoc

```diff
@@ -0,0 +1,73 @@
+
+= Releasing Solr
+:toc: left
+
+== Motivated?
+So you're of the opinion that there are unreleased features or bugfixes committed to the Solr repository that the world needs?
+Are you so convinced of this that you are willing to volunteer to make it happen?
+Good! This document tells you how to get started.
+
+== Overview
+The following is an overview of the artifacts you will be publishing. Although the release wizard should be your primary guide, having a picture of what's going on may help you understand and validate what the release wizard is asking you to do.
+
+There are four major parts of a release. They all become available to the public in slightly different ways, and it helps to understand these differences.
+
+IMPORTANT: All of these publications experience a time delay while background infrastructure detects and propagates your changes. There will be points in the release process when you just need to wait.
+
+=== Source Code and Binaries (downloads.apache.org)
+
+The distribution of a specific version of the source code is the theoretical center of any release by the Apache Software Foundation. As a convenience, precompiled binaries are also provided on downloads.apache.org. The mechanics of this process start with an SVN commit. The result of that commit is automatically synced to the downloads server (~5m?), and then on a longer time frame (6h?) anything on downloads is synced to archives.apache.org.
```

Comment:
Link to https://infra.apache.org/release-publishing.html#timeline which explains the timings.